### ISSUE TITLE
Main navigation keyboard navigation and focus style

### ISSUE THREAD

**Issue by timotheeguerin** [Azure/BatchLabs#1395]
Main navigation(Left bar) doesn't have keyboard navigation for profile. The focus color is also not very visible
